app/views/documents/_form.slim

```diff
@@ -0,0 +1,23 @@
+= form_for @document do |f|
+  .row.justify-content-center.mt-2
+    .col-12.col-sm-8.col-md-6
+      label.form-label Title
+      = f.text_field :title, class: 'form-control'
+
+      - if @document.errors[:title].any?
```

Comment:
@aleguacaran a gem like simple form would've saved you the need to handle error messages manually by input.

Reply:
Yeah, i forgot it, i have worked with formtastic before